contrib/storage-hive/core/src/test/java/org/apache/drill/exec/store/hive/schema/TestColumnListCache.java

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
Could you add a unit test to verify when there are two kind (or more) of columns in e.g. 10 partitions, the physical plan text has only two copies of columns?